client/src/components/NewSongInput.js

```diff
@@ -0,0 +1,26 @@
+import config from "../config";
+import React from "react";
+import PropTypes from "prop-types";
+
+export const NewSongInput = ({link, updateLink, addSong}) => {
+    const checkLinkEntered = (key) => {
+        if (key === config.enterKey) {
+            addSong(link);
+        }
+    };
+    return (
+        <div className="column-center">
+            <input type="text" className="song-link-input font" value={link} autoFocus spellCheck={false}
+                   onKeyDown={(target) => checkLinkEntered(target.key)}
```

Comment:
Those *arrow functions* make the code less readble.
```js
//instead of
const checkLinkEntered = (key) => {
        if (key === config.enterKey) {
            addSong(link);
        }
    };

// do 
const checkLinkEntered = (evnet) => {
        const key = event.key;
        if (key === config.enterKey) {
            addSong(link);
        }
    };
```

Then, is the `return` you code just do:
```js
onKeyDown={checkLinkEntered)
```